plugins/modules/swim_intent.py

```diff
@@ -50,26 +50,26 @@
         type: dict
         suboptions:
           type:
-            description: The source of import, supports url import or local import.
+            description: The source of import, supports remote import or local import.
```

Comment:
Specifies the import source, supporting local import (file) or remote import (URL)

Reply:
specify local or remote as option in playbook